server/controllers/events/events_controller_e2e_test.go

```diff
@@ -42,7 +42,7 @@ import (
 	. "github.com/runatlantis/atlantis/testing"
 )
 
-const ConftestVersion = "0.36.0" // renovate: datasource=github-releases depName=open-policy-agent/conftest
+const ConftestVersion = "0.37.0" // renovate: datasource=github-releases depName=open-policy-agent/conftest
```

Comment:
Could we keep this version always pinned to 0.35.0 or 0.36.0?

Or could we modify the test to not have to check the version? Otherwise we get this chicken-and-egg issue with this dependeny update and the new testing-env image.

Reply:
make sense.

Reply:
rever to 0.36.0